src/include/cpu/power/istep_18.h

```diff
@@ -0,0 +1,634 @@
+#include <cpu/power/scom.h>
+
+void istep_18_11(void);
+void istep_18_12(void);
+
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_0 (38)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_1 (39)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_2 (40)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_3 (41)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_4 (42)
+#define PERV_TOD_ERROR_MASK_REG_RX_TTYPE_5 (43)
+
+#define PERV_TOD_ERROR_REG_RX_TTYPE_2 (40)
+#define PERV_TOD_ERROR_REG_RX_TTYPE_4 (42)
+#define PERV_TOD_ERROR_REG_RX_TTYPE_5 (43)
```

Comment:
These have the same values as above and they are used in close proximity of each other, it would make sense to keep just one of them (shorter).

Reply:
Removed most of the entries, leaving only one